be/src/gutil/linux_syscall_support.h

```diff
@@ -127,8 +127,7 @@
  *      sys_stat64
  *      sys_write
  */
-#ifndef SYS_LINUX_SYSCALL_SUPPORT_H
-#define SYS_LINUX_SYSCALL_SUPPORT_H
```

Comment:
This `endif` need to be removed.

Reply:
There was already a missing endif in that file.  I just re-checked and I still do not see a corresponding endif.

Reply:
If you delete this header guard, the last `#endif` int this file should be deleted.
Or you can skip this file and keep it as before.

Reply:
I have deleted it.